Completion/X/Command/_sublimetext

```diff
@@ -8,6 +8,8 @@ _arguments -s -S : \
   '(-a -n --add --new-window)'{-n,--new-window}'[open new window]' \
   '--project[load specified project]:project file:_files' \
   '(-s --stay)'{-s,--stay}'[keep application activated]' \
+  '--safe-mode[Launch using a sandboxed (clean) environment]' \
```

Comment:
Completion descriptions should not use sentence case (they should start with a lower-case letter). And since the existing descriptions use 'telegraph' style i would make these match (omit 'a' and 'the' where possible)

Reply:
I disagree, but whatever - if those are the rules ...

Reply:
The no-sentence-case thing is mentioned in [the style guide](https://github.com/zsh-users/zsh/blob/master/Etc/completion-style-guide) (under *Descriptions*), and most other functions shipped with zsh follow it. I'm not sure if we have any specific guidance re: telegraph style, but it's certainly common, and if nothing else it's usually a good idea to match a file's existing style

Reply:
> The no-sentence-case thing is mentioned in the style guide (under Descriptions), and most other functions shipped with zsh follow it. ...

Yeah, I got that - hence the new commit.

However, it sounds so ... weird.
And technically, also on the same page it mentions:
> It is a good idea to copy descriptions from the command's man page or --help output.